diagnosticsModule/ADFSDiagnostics/ADFSDiagnostics.psm1

```diff
@@ -0,0 +1,52 @@
+#Requires -Version 4
+#Requires -RunAsAdministrator
+
+<#
```

Comment:
Would it be possible to add some documentation to the README.md to show the basic scenarios, how to use the scripts, etc.? Our bar for documentation is here: https://github.com/Microsoft/adfsOpenSource#-documentation